modules/analysis-common/src/yamlRestTest/resources/rest-api-spec/test/search.query/50_queries_with_synonyms.yml

```diff
@@ -200,3 +200,47 @@
                 query: quick brown
                 operator: and
   - match: { hits.total: 2 }
+---
+# Tests that this returns 400, and not 500 (#123871)
+"Test multi_match phrase with no positions":
```

Comment:
I'm not sure where the best place for this test is...

Reply:
I think it is ok but these tests might run on mixed clusters, you might need to filter those out.

Reply:
Yes, good point